libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -2178,7 +2178,7 @@ export function getParameterFromId(nodeInputs: NodeInputs, parameterId: string):
 export function parameterHasValue(parameter: ParameterInfo): boolean {
   const value = parameter.value;
 
-  if (!isNullOrUndefined(parameter.preservedValue)) {
+  if (!isNullOrUndefined(parameter.preservedValue) && parameter.preservedValue !== '') {
```

Comment:
You can also use !isUndefinedOrEmptyString parameter.preservedValue) the reason being i think null is a valid value.